init-container.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
-# This file is used for locally running the application in a container
-docker build -t wgermany/ladda-"${env:-test}":"${version:-1.0}" . &&
-docker run -v "${path:-$(dirname $(readlink -f $0))/temp}":/app/files -p "${port:-3001}":8080 --env-file ./.env wgermany/ladda-"${env:-test}":"${version:-1.0}"
\ No newline at end of file
+# This is used for locally running the application in a container
+# docker build -t wgermany/ladda-"${env:-test}":"${version:-1.0}" . &&
+# docker run -v "${path:-$(dirname $(readlink -f $0))/temp}":/app/files -p "${port:-3001}":8080 --env-file ./.env wgermany/ladda-"${env:-test}":"${version:-1.0}"
```

Comment:
Potentially worth cleaning up local and prod files so it's less clutter and more clear about how to run the application